docs/getting-started/welcome.md

```diff
@@ -4,39 +4,43 @@ sidebar_label: Welcome to UMA
 slug: /
 ---
 
-:::note uUSD has launched
-Learn how to mint and farm `uUSD` tokens with this [tutorial](users/mint-farm-yusd.md).
+## UMA is open-source infrastructure for deploying and enforcing synthetic assets on Ethereum
 
-Learn how to redeem expired `uUSD` tokens with this [tutorial](users/redeem-tokens.md).
-:::
+UMA enables developers to quickly and easily build synthetic tokens that track the price of anything! 
```

Comment:
We now have similar type of content in three separate places on the docs site. Not necessarily a bad thing, but somewhat redundant, so perhaps we should rethink some of this